### PR TITLE
feat: add text and type to outputs

### PR DESCRIPTION
**Pull request by fampinheiro**

### CODE REVIEW

File: packages/react-components/src/templates/ResearchOutputPage.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  alignSelf: 'stretch',
+  background: pearl.rgb,
+  padding: `${36 / perRem}em ${contentSidePaddingWithNavigation(8)}`,
+});
+
+const articleStyles = css({
+  alignSelf: 'stretch',
+});
+
+const ResearchOutputPage: React.FC<ResearchOutputResponse> = ({
```
Comment (jeysal) [inline packages/react-components/src/templates/ResearchOutputPage.tsx:20]: Quick test to check that the individual parts are rendered?

File: packages/react-components/src/templates/ResearchOutputPage.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  title,
+  text,
+}) => {
+  return (
+    <div css={articleStyles}>
+      <main css={[mainStyles]}>
```
Comment (jeysal) [inline packages/react-components/src/templates/ResearchOutputPage.tsx:26]: `<main>` is already done by the `Layout`, you can collapse the two elements into just the div

File: packages/model/src/research-output.ts [hunk 3]
```diff
@@ -6,7 +8,7 @@ export type ResearchOutputAuthor =
   | { readonly id: string };
 
 export type ResearchOutputFormData = {
-  readonly url: string;
+  readonly url?: string;
```
Comment (jeysal) [inline packages/model/src/research-output.ts:11]: Why this change? In the form design it was a mandatory field

**Review by jeysal — CHANGES_REQUESTED** [review]

File: packages/model/src/research-output.ts (hunk 3 above)
Comment (fampinheiro) [reply packages/model/src/research-output.ts:11]: The proposal doesn't have a URL. But it's also not created by users.
Is keeping the URL required for the form and change it on the ResearchOutputResponse an option?

Comment (jeysal) [reply packages/model/src/research-output.ts:11]: Yeah so this type was specifically for the input form data, I don't think it even reaches the backend before it is re-mapped

**Comment by fampinheiro** [comment]
Merging this and addressing the extraneous comments in my next PR.
@jeysal let me know if I left something out.
